### PR TITLE
INT-1837 Fill "expected" code snippet in GH issue

### PR DESCRIPTION
**Pull request by hbjORbj**
claap: https://app.claap.io/intuita/int-1837-fill-expected-code-snippet-in-gh-issue-via-report-issue-c-BT2DRbCjzO-48B2GAYq7_op

### CODE REVIEW

File: intuita-webview/src/jobDiffView/DiffViewer/DiffItem.tsx [hunk 1]
```diff
@@ -63,7 +71,7 @@ export const JobDiffView = memo(
 					});
 				}
 			}, 1000);
-
+			console.log(newFileContent, originalNewFileContent);
```
Comment (grzpab) [inline intuita-webview/src/jobDiffView/DiffViewer/DiffItem.tsx:74]: please remove this

File: src/selectors/selectSourceControlTabProps.ts [hunk 2]
```diff
@@ -71,23 +71,43 @@ export const selectSourceControlTabProps = (
...
 
-	const { beforeSnippet, afterSnippet } = createBeforeAfterSnippets(
+	const title = `[Codemod:${job.codemodName}] Invalid codemod output`;
+	const { beforeSnippet, afterSnippet: newFileSnippet } =
+		createBeforeAfterSnippets(
+			sourceControlState.oldFileContent,
+			sourceControlState.newFileContent,
+		);
+
+	if (sourceControlState.modifiedFileContent === null) {
```
Comment (grzpab) [inline src/selectors/selectSourceControlTabProps.ts:84]: why this `if`?

File: intuita-webview/src/jobDiffView/DiffViewer/DiffItem.tsx (hunk 1 above)
Comment (hbjORbj) [reply intuita-webview/src/jobDiffView/DiffViewer/DiffItem.tsx:74]: done

File: src/selectors/selectSourceControlTabProps.ts (hunk 2 above)
Comment (hbjORbj) [reply src/selectors/selectSourceControlTabProps.ts:84]: `sourceControlState.modifiedFileContent === null` means that user didn't modify the file. In this case, we don't need to do the following

```
const { afterSnippet: modifiedFileSnippet } = createBeforeAfterSnippets(
		sourceControlState.oldFileContent,
		sourceControlState.modifiedFileContent,
	);
```

**Pull request merged**